src/components/Card/Card.js

```diff
@@ -3,7 +3,7 @@ import './Card.css';
 function Card(props) {
   return (
     <li className='card'>
-      <img className='card__image' src={props.card.image} alt='#'/>
+      <div className={`card__image card__image_type_${props.type}`} style={{backgroundImage: `url(${props.card.image})`}}></div>
```

Comment:
should we specify type of card for classname, since other things like the carousel items i'm making could be understood as "card"? feels like it should be a bit more unique. maybe column-card for example?

Reply:
same would go for component name and any use across the board i suppose.

Reply:
Yes, that's true. I think CarouselCard would be better though. ColumnCard is a bit of a misnomer. Why not RowCard? They are four cards in row too